libs/external-router/src/model/type-route-pair.ts

```diff
@@ -1,17 +1,25 @@
+import { DaffExternalRouterInsertionStrategy } from './insertion-strategy.type';
 import { DaffExternalRouteType } from './route-type';
-import { RouteWithoutPath } from './route-without-path';
+import { DaffRouteWithoutPath } from './route-without-path';
 
 /**
  * A model used to describe the relationship between a {@link DaffExternalRouterType}
  * and its corresponding route.
  *
- * The TypeRoutePair is the pre-provided "glue" that a developer will provide
+ * The DaffTypeRoutePair is the pre-provided "glue" that a developer will provide
  * ahead-of-time so that that the External Router knows what the list of available
  * "types" are (along with that type's corresponding route).
  *
+ * An insertion strategy can be specified if the route needs to be inserted somewhere
```

Comment:
I think it should be mentioned that the default insertion strategy will be the `daffInsertRouteBeforeWildCardStrategy` if none is provided. That's accurate, right?

Reply:
Yes that is correct, I'll add it

Reply:
added